library/src/c_api.rs

```diff
@@ -68,9 +64,10 @@ fn app_config_from_c(c_params: *const AppParameters) -> updater::AppConfig {
 /// configuration compiled into the app.
 #[no_mangle]
 pub extern "C" fn shorebird_init(c_params: *const AppParameters, c_yaml: *const libc::c_char) {
+    print!("in shorebird_init");
```

Comment:
You won't want this for several reasons.
1.  you would want println not print.  print won't add a \n iirc.
2. print won't show up on android, only logs will 😭 
3. this is probably just leftover debugging. :)

Reply:
yeah I was just trying to debug/sanity check wasn't planning to check this in.